utils/nwc.ts

```diff
@@ -205,6 +220,49 @@ async function getNwcConnection(userPubkey: string): Promise<{
   };
 }
 
+export async function getNwcBalance({
+  userPubkey,
+  walletPubkey,
+  nwcRelay,
+}: {
+  userPubkey?: string;
+  walletPubkey?: string;
+  nwcRelay?: string;
+}) {
+  if (!userPubkey || !walletPubkey || !nwcRelay) {
+    return {
+      result: {
+        balance: undefined,
+      },
+    };
+  }
+
+  const { connectionSecret } = await getNwcConnection(userPubkey);
+
+  const requestEvent = await sendNWCRequest({
+    walletPubkey,
+    relay: nwcRelay,
+    method: "get_balance",
+    connectionSecret,
+  });
+  if (!requestEvent) {
+    throw new Error("Failed to send NWC get balance request");
+  }
+
```

Comment:
For future reference?
```suggestion
// Because mutiny's relay is not readable
```